src/themes/_colors.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { FunctionComponent } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const ColourBlocks: FunctionComponent = () => <div className='grid grid-cols-4 grid-rows-1 gap-1 p-2'>
+  <div className='flex h-20 items-center justify-center rounded border border-black bg-primary text-black'>primary</div>
+  <div className='flex h-20 items-center justify-center rounded border border-black bg-primary-dark text-black'>primary-dark</div>
```

Comment:
do we want an alternate text colour for display against primary-dark / use on dark modes?

Reply:
See reply to previous - I was being lazy, I have learnt my lesson :)